cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,50 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+`Release 3.5.5 <http://docs.cask.co/cdap/3.5.5/index.html>`__
+=============================================================
+
+New Features
+------------
+- :cask-issue:`CDAP-4535` - Authentication server announce address is now configurable
+  with the property ``security.auth.server.announce.urls``, which are comma-separated URLs in
+  the form of ``protocol://host:port``. The property
+  ``security.auth.server.announce.address`` is now deprecated. It is only used if it is set
+  but ``security.auth.server.announce.urls`` has not been set.
+  ``security.auth.server.announce.address`` takes a single address in the form of
+  ``host:port`` or ``host``. Protocol and port (if not specified) will be added by the
+  Authentication Server to form the URL. A default URL will be generated by the
+  Authentication Serve if either property is not set.
+
+- :cask-issue:`CDAP-8990` - New configurations have been added to control the YARN
+  application master container memory size, maximum heap memory size, and maximum non-heap
+  memory size: ``twill.java.heap.memory.ratio``, ``twill.yarn.am.memory.mb``, and
+  ``twill.yarn.am.reserved.memory.mb``.
+
+Improvements
+------------
+- :cask-issue:`CDAP-9084` - LogHandler endpoints now returns a 404 status code if the
+  entity (the run id) for which logs are requested does not exist.
+
+Bug Fixes
+---------
+- :cask-issue:`CDAP-9005` - Fixed an issue where ``HBaseQueueDebugger`` failed when
+  authorization was enabled.
+
+- :cask-issue:`CDAP-9160` - Fixed a memory leak issue with the Hadoop FileSystem object.
+
+- :cask-issue:`CDAP-9085` - Fixed an out-of-memory issue for the log saver by adding a
+  limit on the maximum number of events in-memory.
+
+- :cask-issue:`CDAP-8997` - Fixed an issue with uncaught exceptions so that they are
+  logged through the logger, allowing log collections for those exce
```

Comment:
Not a nit. Good catch. Error in original Release Note on confluence (trailing line break that was encoded.)